pkg/spec/createconfig.go

```diff
@@ -145,6 +145,27 @@ func (c *CreateConfig) CreateBlockIO() (*spec.LinuxBlockIO, error) {
 	return c.createBlockIO()
 }
 
+func (c *CreateConfig) AddContainerInitBinary(path string) error {
+	// Add the init binary iff the container will run in a private PID
+	// namespace that is not shared with the host or another pre-existing
+	// container, where an init-like process is already running.
+	if !c.PidMode.IsPrivate() {
+		logrus.Debugf("won't add init binary: PID namespace isn't private")
```

Comment:
This seems like more of an error than a debugf

Reply:
Docker doesn't error out, which made me hesitant to error out.